test/completion.cjs

```diff
@@ -1102,6 +1149,24 @@ describe('Completion', () => {
       r.logs[0].should.match(/ndm --get-yargs-completions/);
     });
 
+    it('uses a "safe" app name for the bash completion function name', () => {
+      process.env.SHELL = '/bin/zsh';
+      const r = checkUsage(() =>
+        yargs([]).scriptName('1 test ing').showCompletionScript()
+      );
+
+      r.logs[0].should.match(/_1_test_ing/);
+    });
+
+    it('generates multi-word completion handler', () => {
+      process.env.SHELL = '/bin/zsh';
+      const r = checkUsage(() =>
+        yargs([]).scriptName('1 test ing').showCompletionScript()
+      );
+      console.log(r.logs[0]);
+      r.logs[0].should.match(/_yargs_completions "1=test=ing"/);
+    });
+
```

Comment:
These 2 are in the pre-existing zsh `describe`. They are just ensuring multi word support happens as expected.